Index: electrum-dash-post-3.2.2/electrum-dash
===================================================================
--- electrum-dash-post-3.2.2.orig/electrum-dash
+++ electrum-dash-post-3.2.2/electrum-dash
@@ -28,6 +28,8 @@ import sys
 import warnings
 
 
+os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION']='python'
+
 MIN_PYTHON_VERSION = "3.6.1"  # FIXME duplicated from setup.py
 _min_python_version_tuple = tuple(map(int, (MIN_PYTHON_VERSION.split("."))))
 
