--- buildozer/targets/android.py.orig	2020-04-28 15:49:20.583469667 +0000
+++ buildozer/targets/android.py	2020-04-28 15:08:38.450603953 +0000
@@ -915,6 +915,9 @@
             ("--ndk-api", config.getdefault('app', 'android.minapi',
                                             self.android_minapi)),
         ]
+        apk_version_code = environ.get('APK_VERSION_CODE', None)
+        if apk_version_code is not None:
+            build_cmd += [("--numeric-version", apk_version_code)]
         is_private_storage = config.getbooldefault(
             'app', 'android.private_storage', True)
         if is_private_storage:
